aggregator_api/src/routes.rs

```diff
@@ -456,7 +456,8 @@ pub(super) async fn post_taskprov_peer_aggregator<C: Clock>(
         req.tolerable_clock_skew,
         req.aggregator_auth_tokens,
         req.collector_auth_tokens,
-    );
+    )
+    .map_err(|e| Error::BadRequest(format!("Invalid request: {e}")))?;
```

Comment:
super-nit: since this is `Error::BadRequest` anyway, I suspect the resulting error and/or HTTP problem doc will already say bad or invalid request and it's not necessary to repeat that in the format string.

Reply:
The aggregator API does not use HTTP problem documents, it just uses this string as the entirety of the response body.